pyproject.toml

```diff
@@ -148,3 +148,15 @@ isort.known-first-party = ["deeprank2"]
 ]
 "docs/*" = ["ALL"]
 "tests/perf/*" = ["T201"] # Use of print statements
+
+[tool.bumpversion]
```

Comment:
How does it handles the different syntaxes? For example, `version: "3.0.4"` of the CITATION.cff file versus the `__version__ = "3.0.4"` of the init

Reply:
I don't know how bump-my-version handles those differences (you'd need to check their documentation), but I have tested that it works as expected.
Good thing that you flagged this, though, because the current version listed was incorrect 😆